Tools/TestWebKitAPI/Tests/WebKitCocoa/ScreenTime.mm

```diff
@@ -291,7 +299,7 @@ static BOOL systemScreenTimeBlockingViewIsPresent(TestWKWebView *webView)
         PAL::getSTWebpageControllerClass(),
         @selector(setProfileIdentifier:),
         imp_implementationWithBlock(^(id object, NSString *profileIdentifier) {
-            identifier = profileIdentifier;
+            identifier = [profileIdentifier copy];
```

Comment:
It would be better to simply make `identifier` a `RetainPtr`, since the `copy` here on its own is a memory leak.